src/doc/en/developer/coding_in_cython.rst

```diff
@@ -32,13 +32,13 @@ up-to-date information or check out the
 to get started immediately.
 
 
-Writing cython code in Sage
+Writing Cython code in Sage
 ===========================
 
 There are several ways to create and build Cython code in Sage.
 
-#. In the Sage Notebook, begin any cell with ``%cython``. When you
-   evaluate that cell,
+#. In the Sage Notebook or the Sage command line, begin any cell with
+   a line containing ``%%cython``. When you evaluate that cell,
 
```

Comment:
I think `the Sage notebook` is appropriate since `Sage Notebook` refers to the old deprecated Sage's own notebook. Now there is only Jupyter notebook with Sage kernel.

Reply:
You mean is not appropriate (?) 

Anyway looking through the documentation it doesn't seem to have a special name, it's just referred to by "the notebook interface".

Maybe I can just write "In the notebook interface or the command-line interface, begin any cell with a line…"

Problem (though this is maybe too nitpicky: I'm not sure what a `sage: ` part in the command-line is called. Is "cell" appropriate?

The terminology isn't consistent throughout either. https://doc.sagemath.org/html/en/tutorial/interactive_shell.html uses "interactive shell", and  https://doc.sagemath.org/html/en/reference/repl/index.html uses "command line".

Reply:
I meant that "the Sage Notebook" (with capital `N`) is a proper noun, and hence it implies there is some Notebook software that we developed. But there is no such thing after we dropped the classic Sage Notebook. So I suggested "the Sage notebook" (with small `n`) that rhymes well with "the Sage command line".

You may just leave it as it is, if you are reluctant to touch things out of the scope of your PR.

"sage: " is called the Sage prompt, if your question means that.

Reply:
Might as well fix. Found a few other instances.

(though some other instances are from other languages, I'm not sure if it's appropriate to change it because some languages may use capitalization in other ways than English e.g. if I recalled correctly German has some capitalization rules.

Reply:
Thanks.